src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -144,8 +144,9 @@ private void saveResponse(FeedbackResponseAttributes response)
             throws EntityDoesNotExistException {
         if (response.getId() != null) {
             // Delete away response if any empty fields
-            if (response.responseMetaData.getValue().isEmpty() || 
-                response.recipientEmail.isEmpty()) {
+            if (response.responseMetaData.getValue().isEmpty() ||
+                    response.recipientEmail.isEmpty() ||
+                    isEmptyContributionResponse(response)) {
```

Comment:
Instead of doing this, I think we can override isQuestionSkipped in FeedbackContributionQuestionDetails instead. That's called earlier in this action and sets the answer to be empty so this branch is taken.